modules/serverless-ossm-setup-without-kourier-mtls.adoc

```diff
@@ -0,0 +1,199 @@
+[id="serverless-ossm-setup-without-kourier-mtls_{context}"]
+= Integrating {ProductShortName} and {ServerlessProductName} with mTLS enabled
+
+.Procedure
+
+. Create a `ServiceMeshControlPlane` object in the `istio-system` namespace that has mTLS enabled.
+. Add the namespaces that you would like to integrate with {ProductShortName} to the `ServiceMeshMemberRoll` object as members:
++
+[source,yaml]
+----
+apiVersion: maistra.io/v1
+kind: ServiceMeshMemberRoll
+metadata:
+  name: default
+  namespace: istio-system
+spec:
+  members: <1>
+    - knative-serving
+    - <namespace>
+----
+<1> A list of namespaces to be integrated with {ProductShortName}.
++
+[IMPORTANT]
+====
+This list of namespaces must include the `knative-serving` namespace.
+====
+. Apply the `ServiceMeshMemberRoll` resource:
++
+[source,terminal]
+----
+$ oc apply -f <filename>
+----
+. Create the necessary gateways so that {ProductShortName} accepts traffic:
++
+[source,yaml]
+----
+apiVersion: networking.istio.io/v1alpha3
+kind: Gateway
+metadata:
+  name: knative-ingress-gateway
+  namespace: knative-serving
+spec:
+  selector:
+    istio: ingressgateway
+  servers:
+    - port:
+        number: 443
+        name: https
+        protocol: HTTPS
+      hosts:
+        - "*"
+      tls:
+        mode: SIMPLE
+        credentialName: <wildcard_certs> <1>
+---
+apiVersion: networking.istio.io/v1alpha3
+kind: Gateway
+metadata:
+ name: knative-local-gateway
+ namespace: knative-serving
+spec:
+ selector:
+   istio: ingressgateway
+ servers:
+   - port:
+       number: 8081
+       name: http
+       protocol: HTTP <2>
+     hosts:
+       - "*"
+---
+apiVersion: v1
+kind: Service
+metadata:
+ name: knative-local-gateway
+ namespace: istio-system
+ labels:
+   experimental.istio.io/disable-gateway-port-translation: "true"
+spec:
+ type: ClusterIP
+ selector:
+   istio: ingressgateway
+ ports:
+   - name: http2
+     port: 80
+     targetPort: 8081
+----
+<1> Add the name of your wildcard certificate.
+<2> The `knative-local-gateway` serves HTTP traffic. That means that traffic coming from outside of {ProductShortName}, but using an internal hostname, such as `example.default.svc.cluster.local` is not encrypted. You can set up encryption for this path by creating another wildcard certificate and an additional gateway that uses a different `protocol` spec.
++
+.Example `knative-local-gateway` using HTTPS
+[source,yaml]
+----
+apiVersion: networking.istio.io/v1alpha3
+kind: Gateway
+metadata:
+  name: knative-local-gateway
+  namespace: knative-serving
+spec:
+  selector:
+    istio: ingressgateway
+  servers:
+    - port:
+        number: 443
+        name: https
+        protocol: HTTPS
+      hosts:
+        - "*"
+      tls:
+        mode: SIMPLE
+        credentialName: <wildcard_certs>
```

Comment:
```suggestion
        credentialName: <wildcard-certs>
```
Better align with example openssl command above.

Reply:
same as above, it needs to be an underscore